### ISSUE TITLE
GUI: LSL Sends Corrupt Data When Filters On

### ISSUE THREAD

**Issue by coder1923** [OpenBCI/OpenBCI_GUI#880]
## Problem

When Filters are toggled on during LSL Networking, the data sent out is corrupt.
This is only noticeable when using Cyton, and NOT when using Synthetic data.
I tested this extensively, many times using different filters.

## Expected 

Expect that data sent over LSL is exactly as what is visualized on the GUI, when Filters are toggled ON.
Rather, currently, the data is completely corrupt.

## Operating System and Version

Windows

## GUI Version

5.0.0

## Running standalone app

Running the downloaded App.

## Type of OpenBCI Board

Cyton.

## Are you using a WiFi Shield?

No, using dongle.

**Comment by retiutut** [comment]
There is no supporting data for this claim. Please provide evidence.

**Comment by coder1923** [comment]
@retiutut Please check this video I recorded.

https://youtu.be/cL470OmWCNg

As you can see the data on the receiving end is not the same as the data as displayed in the GUI.
Furthermore, even after applying Notch filters to the data on the receiving end, it is different from the one displayed in the GUI.

If I'm not mistaken, the "Filters On" toggle in the GUI is supposed to send data as is displayed on GUI after filters applied, correct?

This is the issue. You can reproduce it by turning on the filters toggle and sending data over LSL. The data being sent, as I'm observing is not filtered.

**Comment by coder1923** [comment]
@retiutut I think the issue stems when either the 50Hz toggle is on... or when the Filter toggle is On.
I can't try it on 60Hz in the country I'm in.

**Comment by retiutut** [comment]
I was able to use an LSL stream viewer on Windows to see that what you are saying is entirely valid. A screenshot could have helped see the "corrupt data".

After going over the entire data pipeline to investigate LSL on the whole, I discovered a simple coding error that must have come about when we refactored the GUI for v5 from v4.

A fix is about to merge to the development branch. https://github.com/OpenBCI/OpenBCI_GUI/pull/879

Thank you for mentioning that something felt off about LSL streaming, which led to the investigation and fix.
